CONTRIBUTING.md

```diff
@@ -86,4 +86,6 @@ There are a few goals of `clap` that I'd like to maintain throughout contributio
   - This is also true of generating help and usage information (although *slightly* less stringent, as the program is about to exit)
 * Try to be cognizant of memory usage
   - Once parsing is complete, the memory footprint of `clap` should be low since the main program is the star of the show
-* `panic!` on *developer* error, exit gracefully on *end-user* error
+* `panic!` on *developer* error
```

Comment:
I don't know if linking to a file is a good idea because it's easy for these links to silently break. If we're going to link we should probably use github permalinks to be safe(r). I don't know how others feel about this though.

Reply:
I have no strong preference, so I'll update to that.